common/src/main/java/com/rln/client/damlClient/RLNDamlClient.java

```diff
@@ -186,6 +189,34 @@ public void archiveBalance(ArchiveBalanceParameters parameters) {
         commandPublisher.onNext(new ClientCommand(event, update, parameters.getProvider(), parameters));
     }
 
+    @Override
+    public void changeBalance(ChangeBalanceParameters parameters) {
+        var event = String.format("Changing balance %s (provider: %s)", parameters.getIban(), parameters.getProvider());
+        if (parameters.getChange().compareTo(BigDecimal.ZERO) == 0) {
+          return;
+        }
+        Update<?> update;
+        if (parameters.getChange().compareTo(BigDecimal.ZERO) < 0) {
```

Comment:
**Minor:** this is just an idea. I think it would read better if we extracted these to something like:

```java
public class BigDecimals {
  public static boolean equals(BigDecimal x, BigDecimal y) {
    return 0 == x.compareTo(y);
  }
  public static boolean lessThan(BigDecimal x, BigDecimal y) {
    return x.compareTo(y) < 0;
  }
}
```